easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.0-2023b.yml

```diff
@@ -8,3 +8,33 @@ easyconfigs:
   - matplotlib-3.8.2-gfbf-2023b.eb:
       options:
         from-pr: 19552
+  - AOFlagger-3.4.0-foss-2023b.eb:
+      options:
+        from-pr: 19840
+        # Exclude Lua from `filter-deps` as the compat layer version is too old for the software
+        filter-deps:
+          - Autoconf
+          - Automake
+          - Autotools
+          - binutils
+          - bzip2
+          - DBus
+          - flex
+          - gettext
+          - gperf
+          - help2man
+          - intltool
+          - libreadline
+          - libtool
+          - ncurses
+          - M4
+          - makeinfo
+          - util-linux
+          - XZ
+          - zlib
```

Comment:
This shouldn't be required anymore, as we no longer filter Lua in the EasyBuild configuration, see https://github.com/EESSI/software-layer/blob/2023.06-software.eessi.io/configure_easybuild#L28.

```suggestion
```

Reply:
Great! Then I'll leave out AOFlagger entirely because it is a dependency for the other two.

Reply:
I think that is actually still required, as otherwise EasyBuild will not be able to pick up the dependencies from that PR (usually it does automatically do that, but not in combination with easystack files). I remembered we had an open issue about it (see https://github.com/EESSI/software-layer/issues/378), and I just noticed it was actually created because of one of your previous PRs :wink:

In this case it may be even worse, since you had to bump the version of quite a few other dependencies in that same PRs, which means that all of them would have to be added to the easystack file as well... So we definitely should come up with a solution for this.

Reply:
Oh dear, my bad gentlemen... I guess my mind said "eh, that thing is merged, should be fine now", but I didn't realize it didn't make it into a release yet.

I'm in favour of listing those explicitely in the buildlist and using `--include-easyblocks-from-pr`. It does not make sense to modify the current EasyConfig PR, which targets `develop`, and which will be released simultaneously with the EasyBlock PR. I.e. for that PR, it makes total sense not to include the hardcoded Python path in the `configopts` anymore...

Reply:
Yep, I fully agree!

Reply:
> I think that is actually still required, as otherwise EasyBuild will not be able to pick up the dependencies from that PR (usually it does automatically do that, but not in combination with easystack files). I remembered we had an open issue about it (see #378), and I just noticed it was actually created because of one of your previous PRs 😉
> 
> In this case it may be even worse, since you had to bump the version of quite a few other dependencies in that same PRs, which means that all of them would have to be added to the easystack file as well... So we definitely should come up with a solution for this.

If I remember correctly, the issue with #378 was that the dependencies for the easyconfigs had already been merged and part of an EasyBuild release. 
But afterwards, we discovered the Python path issue, so I created a new PR with the Python path in the configopts. So this time around, there were easyconfigs for the same software version + toolchain in the new PR and EasyBuild. Turns out the dependencies in EasyBuild are selected before any updated dependencies in the PR. So it seems to be an issue of search order.